tools/webidl_binder.py

```diff
@@ -59,6 +59,8 @@ def getExtendedAttribute(self, _name):
 parser = argparse.ArgumentParser()
 parser.add_argument('--wasm64', action='store_true', default=False,
                     help='Build for wasm64')
+parser.add_argument('--thread_safe', action='store_true', default=False,
```

Comment:
can you use dashes in the option name (e.g. "--thread-safe" )

Reply:
Fixed.